src/user-research/how-we-do-user-research.md

```diff
@@ -10,9 +10,9 @@ We have a set of guides that explain how we do user research at dxw.
 The guidance builds on our user research [principles](/user-research/#user-research-principles). It also builds on the [guidance for user research in the GOV.UK Service Manual](https://www.gov.uk/service-manual/user-research)
 so we can help teams meet the Government [Service Standard](https://www.gov.uk/service-manual/service-standard).
 
-Note that this guidance applies to all dxw staff, dxw friends, and partner and client staff doing research activities, like interviews, visits, surveys and tests. And particularly the guidance on [Doing research safely](/user-research/doing-research-safely/), [Getting informed consent](/user-research/getting-informed-consent-for-user-research/), [Protecting participant privacy](/user-research/protecting-participant-privacy/) and [Taking care of research data](/user-research/taking-care-of-research-data/).
+Note that this guidance applies to all dxw staff, dxw friends, and partner and client staff doing research activities, like interviews, visits, surveys and tests. And particularly the guidance on [Doing research safely](/user-research/doing-research-safely/), [Getting informed consent](/user-research/getting-informed-consent-for-user-research/), [Protecting participant privacy](/user-research/recruiting-participants-and-protecting-privacy/) and [Taking care of research data](/user-research/taking-care-of-research-data/).
```

Comment:
Should "Protecting participant privacy" be updated to the new page name now?

Reply:
I've linked this to the specific section on privacy